ebs-janitor/config.go

```diff
@@ -29,7 +32,7 @@ func init() {
 	viper.SetEnvPrefix("janitor")
 
 	defaults := map[string]interface{}{
-		"debug":           true,
+		"debug":           false,
```

Comment:
hmmm, I left this true for the purpose of the local development to avoid any bugs and delete anything critical

Reply:
FOr some reason though having this true during unit testing when I tried to handle the Debug value through the environment variable dryRun was ignoring the changes. For this reason made this false and also did the follow-up change you commented below.